spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project){ create(:project) }
+  let(:task){ create(:task, project: project) }
+  let(:complete_task){ create(:task, :complete, project: project) }
+
   describe 'Task一覧' do
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
         # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
         # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
         visit project_path(project)
         click_link 'View Todos'
+        windows = page.driver.browser.window_handles
+        page.driver.browser.switch_to.window(windows.last)
```

Comment:
修正としては問題ないですが、1行でも記載できることは知っておいてください。
```suggestion
        switch_to_window(windows.last)
```

Reply:
[54682c2](https://github.com/yoshiki105/rspec_app_exam/pull/1/commits/54682c26880a68858b96a57442a0f7b2b4140ab5) で簡潔な表記に直しました。